crates/vault-registry/src/lib.rs

```diff
@@ -1683,6 +1718,25 @@ impl<T: Config> Pallet<T> {
             .ok_or(Error::<T>::TryIntoIntError)?;
         Ok(signed_fixed_point)
     }
+
+    pub(crate) fn percentage_of_amount(
```

Comment:
Percentage is a little bit misleading. What do you think about this?
```suggestion
    pub(crate) fn fraction_of_amount(
```